src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -3565,12 +3575,12 @@ namespace HVACUnitaryBypassVAV {
 
             Real64 CpSupplyAir = Psychrometrics::PsyCpAirFnWTdb(OutAirHumRat, OutAirTemp);
 
-            //     Find the supply air temperature that will force the box to full flow
+            // Find the supply air temperature that will force the box to full flow
             if (BoxOutletNodeNum > 0) {
-                if (CpSupplyAir * DataLoopNode::Node(BoxOutletNodeNum).MassFlowRateMax == 0.0) {
+                if (DataLoopNode::Node(BoxOutletNodeNum).MassFlowRateMax == 0.0) {
```

Comment:
Don't need CpAir in the conditional, it's just a constant. If Mdot = 0, then CpAir * Mdot also = 0 so don't need CpAir here

Reply:
What if mdot is greater than zero but cp is zero?

🤣 🤣 🤣